.eslintrc.json

```diff
@@ -19,7 +19,7 @@
   },
   "plugins": ["react"],
   "rules": {
-    "react/jsx-filename-extension": ["warn", {"extensions": [".js", ".jsx"]}],
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
```

Comment:
Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/react-redux).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))


If there's a better reason of using this config, please open a PR on the linter config repo.

Reply:
Thank you @bolah2009 for your review.

That rule helps to ``remove`` the ``()`` for single-function arguments

<code>
   const greeting = name => <b>{name}</b>
</code>
<br><br>
<i> Code here produces error </i><br>
<code>
     const greeting = (name) => <b>{name}</b>
</code>

<br><br>
What do you think? Does it require PR on the linter repo? Although turning it ``off`` could unintentionally allow some really ugly code slip through if the developer does not remember.

Reply:
What are the examples of this `ugly code`? :thinking: 

I had a similar discussion with another student about this too.

Reply:
Well, let's say the word ``Ugly`` is relative :smile: 

In the example above. removing parenthesis () from the ``name`` argument kind of reduces markups and prints on the screen, compared to the code example 2.

In the end, I think it's about preference or personal style

Reply:
You right, can you still open a PR. I think you should use ESLint doc as reference

Reply:
Okay. Thanks.

Reply:
Here https://eslint.org/docs/rules/arrow-parens#further-reading and particularly this https://github.com/airbnb/javascript#arrows--one-arg-parens

Reply:
@bolah2009, now, it seems like I am in a fix. :confused: 

Airbnb standard thinks it's good practice to have the () **"arrow-parens"** set to **"always"** on the other hand, Eslint was inspired by Airbnb to add support for the "as-needed"

And based on our discussion so far, its good to use both syntaxes **as needed**, rather than turning the rule ``off``. I believe this is the standpoint, for the PR on Microverse repo. 
Hope I am on the right track?

Reply:
You are!

Reply:
Hi @bolah2009,
I have created, the PR [here](https://github.com/microverseinc/linters-config/pull/49)

Reply:
Seen, thanks!